src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
+	 *
+	 * @since TBD
+	 */
+	public function enable(  ) {
+		if ( has_action( 'tribe_log', [ $this, 'dispatch_log' ] ) ) {
+			return;
+		}
+
+		remove_action( 'tribe_log', [ $this, 'dispatch_log' ] );
+	}
+
+	/**
+	 * Disables the logging functions.
+	 *
+	 * @since TBD
+	 */
+	public function disable() {
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Opening brace should be on a new line

<sub>hash: d66b682696da725e481ed0792f1d48c0</sub>